source/transactions/tests/README.rst

```diff
@@ -86,7 +86,15 @@ Each YAML file has the following keys:
   - ``skipReason``: Optional, string describing why this test should be
     skipped.
 
-  - ``clientOptions``: Optional, parameters to pass to MongoClient().
+  - ``topology``: Optional. An array of server topologies against which to run
+    the test. Valid topologies are "replicaset" and "sharded". The default is
+    ["replicaset", "sharded"].
```

Comment:
Oy, I missed that retryable reads added `topology` at the per-operation level. I don't see anything in that README that talks about a default value, though, but I suppose that's a separate question for Vincent.

Is it feasible for both specs to keep this at the top-level? I was talking to @prashantmital about this only yesterday and explaining why I thought min/max server versions and topologies were suitable for top-level fields, as it cuts down on repeated information within the `tests[i]` document and encourages spec authors to split up their test files. Assuming it's possible to get retryable reads on board as well, while that is still being reviewed, I'd like to see this changed instead of going all-in on the change stream spec's format -- from what I saw, it was only used there because standalone and replica sets error tests were placed in the same file.

Reply:
I'm happy to move ``topology`` to the top-level for retryable reads. Good catch about the test plan README, I'll update it now. The default is currently ``["single", "replicaset", "sharded"]``

Reply:
I also promoted this to the top-level and made the default `["single", "replicaset", "sharded"]`.